src/middlewared/middlewared/plugins/filesystem.py

```diff
@@ -206,7 +206,7 @@ def stat(self, path):
     @private
     @accepts(
         Str('path'),
-        Str('content', max_length=512000),
+        Str('content', max_length=2048000),
```

Comment:
We do transfers in chunks, any sane limit will work, what is the reason for this?

Reply:
Because thats the limit on the method size. We dont want a client sending a payload of 500MB in a single method call message.

Reply:
I mean what is the reason of increasing this limit?

Reply:
I want to make the transfer in a single call. Splitting it in 3-4 chunks will make it slower